clients/admin-ui/src/features/common/table/v2/filters/GlobalFilterV2.tsx

```diff
@@ -32,6 +32,7 @@ export const GlobalFilterV2 = ({
     if (!value) {
       onClear();
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
I think this not related to your changes, but still we should be able to fix it by including onClear on the deps.
```suggestion

```